### PR TITLE
fix position of the spinner on the personal settings

### LINKED ISSUE #2198: Don’t show spinner in personal settings content
In the personal settings, there’s a spinner shown in the center of view. This is confusing since the elements in the viewport area already loaded anyway, and the spinner is not needed.
It actually makes the software seem slower than it is in this case. Can be removed.

cc @nextcloud/javascript

### PR DESCRIPTION
**Pull request by schiessle**
Fix positioning of the spinner in Chromium for the personal settings. The spinner was shown in the middle of the profile information of the user. Now we show it in the section where it belongs.

fix https://github.com/nextcloud/server/issues/2198

cc @jancborchardt this is what we observed last week in Stuttgart

### CODE REVIEW

**Comment by LukasReschke** [comment]
>  the page is loaded fast enough, therefore I removed it completely.

Did you test that also on a remote server where there is the roundtrip of another TCP connection?

cc @ChristophWurst

File: settings/templates/personal.php [hunk 1]
```diff
@@ -321,8 +321,6 @@ class="password-confirm-required"
 				<th></th>
 			</tr>
 		</thead>
-		<tbody class="token-list icon-loading">
-		</tbody>
```
Comment (ChristophWurst) [inline settings/templates/personal.php]: 👎 I'm pretty sure you just removed the list of browser sessions which is attached via js on page load, see https://github.com/nextcloud/server/blob/206ef51eac4f65242658733f6899f7fe3fa473b2/settings/js/authtoken_view.js#L77-L96 🙈

File: settings/templates/personal.php [hunk 2]
```diff
@@ -337,8 +335,6 @@ class="password-confirm-required"
 				<th></th>
 			</tr>
 		</thead>
-		<tbody class="token-list icon-loading">
```
Comment (ChristophWurst) [inline settings/templates/personal.php]: same applies for the list of device tokens. the spinner actually represents the loading state of the table - the data is fetched *after* the page has loaded. Maybe the spinner is just not properly positioned?

**Review by ChristophWurst — DISMISSED** [review]

**Comment by ChristophWurst** [comment]
> Did you test that also on a remote server where there is the overhead of the roundtrip of another TCP connection?

Right. I remember adding that spinner because it took noticeable time to load the list of browsers/clients when using the php built-in server. Users with slow internet connections or slow servers might experience the same delay. Therefore I'm not a big fan of removing this.

**Comment by jancborchardt** [comment]
> I remember adding that spinner because it took noticeable time to load the list of browsers/clients when using the php built-in server.

This list is out of the normal viewport though, so most people who don’t scroll down fast enough are likely to never notice. Only this list should get a spinner then, not the whole content area.

**Comment by schiessle** [comment]
will have a look tomorrow

**Comment by MorrisJobke** [comment]
> I think also the other sections doesn't need a spinner, the page is loaded fast enough, therefore I removed it completely.

This is not really an excuse - what if you are on a slow connection - giving no feedback at all is not really good UX.

**Comment by schiessle** [comment]
Should be fixed now... btw, this was only broken for Chromium. I tested it now with Chromium and Firefox.

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Comment by ChristophWurst** [comment]
👎 this breaks the spinner position on other places like the settings menu's spinners: bildschirmfoto von 2016-11-22 11-45-08

Maybe we should apply the CSS fixes only specifically for those two occurrences on the personal settings page?

**Comment by MorrisJobke** [comment]
> 👎 this breaks the spinner position on other places like the settings menu's spinners:

but in the sharing sidebar the spinner works

**Comment by MorrisJobke** [comment]
I will look into this after lunch

**Comment by schiessle** [comment]
my last commit should fix the spinner position in the top-right menu, @ChristophWurst please have a look

**Comment by skjnldsv** [comment]
Please don't touch the spinner! 
It has been a pain in the ass to work on this and finally decide what tech to choose. Everythings work fine now. Except for the table which doesn't allow relative positioning.

Like I said in #2198, 
> We should not use table, this should do the trick

**Comment by MorrisJobke** [comment]
As discussed with @skjnldsv on IRC - @skjnldsv will take over this one.

**Comment by MorrisJobke** [comment]
Superseeded by #2316

**Pull request closed without merging**